src/stores/datStore.ts

```diff
@@ -84,6 +84,8 @@ export interface SellReturn {
 
 export type TradeEvent = BuyEvent | SellEvent;
 
+export const TRADES_FROM_LAST_BLOCKS = 1000;
```

Comment:
Should this be named `BLOCKS_TO_GET_TRADES_FROM` or `BLOCK_FETCH_NUMBER` or maybe `FROM_BLOCK_OFFSET`